ffi/opaque/README.md

```diff
@@ -0,0 +1,30 @@
+# Opaque example
+
+This sample demonstrates use of Opaque types between C and Dart.
+
+While ffi has good support for dealing with common Struct/struct between code in the two languages, sometimes Dart code
```

Comment:
I'd write FFI, not ffi. It is an initialism after all. Also, please wrap `Struct` and other code in backticks.